src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+$cell-size: 100px;
+$calendar-size: ($cell-size + 1) * 7;
+$calendar-gap: 1px;
+$calendar-padding: 10px;
+$number-of-columns: 7;
+$days: ((mon, 1), (tue, 2), (wed, 3), (thu, 4), (fri, 5), (sat, 6), (sun, 7));
+
+@mixin center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@keyframes move {
+  0% {
+    transform: 0;
```

Comment:
The `transform` property should be specified as `transform: translateY(0);` instead of `transform: 0;` to ensure it is valid CSS.